### PR TITLE
NRS-5--Input-data-validator

### PR DESCRIPTION
**Pull request by MolochkovSA**

### CODE REVIEW

File: users/validatior.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Joi from 'joi'
+
+const sexValues = { male: 'male', female: 'female' }
```
Comment (DmitriyMolochkov) [inline users/validatior.js]: Export this from constants file

File: users/validatior.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+function userValidation(req, res, next) {
+  const { error, value } = schema.validate(req.body)
+  if (error) {
+    res.status(418).send(error.details[0].message)
+  } else {
+    req.body = value
+    next()
+  }
+}
```
Comment (DmitriyMolochkov) [inline users/validatior.js]:
```suggestion
export function ValidationError (res, errors) {
  const body = {
    code: 'VALIDATION_ERROR',
    params: errors
  };
  res.status(400).send(body);
} 
function userValidation(req, res, next) {
  const { error, value } = schema.validate(req.body)
  if (error) {
    return ValidationError(res, error)
  } else {
    req.body = value
    next()
  }
}
```

Comment (MolochkovSA) [reply users/validatior.js]: Done

File: users/validatior.js (hunk 1 above)
Comment (MolochkovSA) [reply users/validatior.js]: Done

**Review by DmitriyMolochkov — APPROVED** [review]

**Pull request merged**